### DISCUSSION TITLE
set loop="0" as default

### DISCUSSION THREAD

**Discussion by joachimheintz** [brunoherbelin/vimix#103] [Q&A] [answered]
i have a number of video files which i want to play without loop.
i saved the session and then went into the xml file replacing all `loop="1"` by `loop="0"`.
is there a better way to do this which i did not find?

and a bit similar, i did not find how to show/hide the log window.

thanks -
  joachim

**Comment by brunoherbelin** [comment, marked as answer]
<img width="462" alt="image" src="https://github.com/brunoherbelin/vimix/assets/60393189/db01e2ee-7462-4174-8e11-bf530298493f">

You can change the loop / bounce / stop behavior of video in the player window.

For the log window, the closing X icon is on the top-right corner. 
(this window is indeed a bit different from the others, but also because it is more for debug...)

**Comment by brunoherbelin** [comment]
![image](https://github.com/brunoherbelin/vimix/assets/60393189/aea44f38-4694-4139-ab69-1bad1fd7a99e)
NB: the new 'View' menu in version 0.8.1 now explicitely includes the 'Logs' window selection.
